core/src/main/java/io/varhttp/BaseVarConfigurationContext.java

```diff
@@ -70,4 +72,9 @@ ObjectFactory getObjectFactory() {
 	public ControllerExecution getNotFoundController() {
 		return notFoundController;
 	}
+
+	@Override
+	public BiConsumer<String, Method> getOnControllerAdd() {
```

Comment:
Having a dedicated interface instead of BiConsumer would make it more explanatory

Reply:
d2fbbe1